src/content/docs/browser/browser-monitoring/troubleshooting/troubleshoot-your-browser-monitoring-installation.mdx

```diff
@@ -69,30 +69,12 @@ These troubleshooting steps apply to problems when the browser monitoring agent
 
     ```html
     <script type="text/javascript">
-        (window.NREUM||(NREUM={})).loader_config={xpid:"VRUGVVJS";window.NREUM||(NREUM={}),__nr_require=function a(b,c,d){ ...
+        ;window.NREUM||(NREUM={});NREUM.init={ ...
```

Comment:
@jeff-colucci Is this JavaScript example incomplete on purpose?

Reply:
This seems to be the correct example based on the audit doc I got, the PR has been reviewed by the SMEs and it didn't come up but I will double check just to make sure before publishing